keps/sig-node/4580-deprecate-kubelet-runonce/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: Deprecate & remove Kubelet RunOnce mode
+kep-number: 4580
+authors:
+  - "@HirazawaUi"
+owning-sig: sig-node
+participating-sigs:
+status: provisional
+creation-date: 2024-04-17
+reviewers:
+  - "@tallclair"
+  - "@dchen1107"
+approvers:
+  - TBD
+
+see-also:
+
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
```

Comment:
I'd expect to see
```suggestion
stage: deprecated
```

Reply:
semms better.

Reply:
> I'm not sure there is any precedent for introducing deprecated gate in KEP

The other way to do it is to introduce a feature gate named legacy something, eg `LegacyNodeRoleBehavior`. Doing it that way, `LegacyNodeRunOnceMode` (or similar) would come in as an alpha, off-by-default feature gate that is later deprecated and finally removed.

There are some examples in https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates-removed/

Reply:
Thanks, seems better.